vmdb/spec/models/provider_foreman_spec.rb

```diff
@@ -0,0 +1,36 @@
+require "spec_helper"
+
+describe ProviderForeman do
+  let(:provider) do
+    FactoryGirl.build(:foreman_provider,
+                      :url      => "example.com",
```

Comment:
This can be defaulted in the factory as well.  In addition, if you are defaulting userid and password, can the test just use those default values instead of overriding?